project/MiMa.scala

```diff
@@ -744,6 +744,10 @@ object MiMa extends AutoPlugin {
         ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.recoverWithRetries")
       ),
       "2.4.4" -> Seq(
+        // #20371, missing method and typo in another one making it impossible to use HTTPs via setting default HttpsConnectionContext
+        ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.http.scaladsl.HttpExt.setDefaultClientHttpsContext"),
```

Comment:
The meth may have unintended side effects... yeah. – oh wait, method! ;-)

So the story here is that it was completely broken for everyone - if you called this method you likely did not get what you wanted.